src/overlays/actors/ovl_Bg_Goron_Oyu/z_bg_goron_oyu.c

```diff
@@ -23,26 +21,133 @@ const ActorInit Bg_Goron_Oyu_InitVars = {
 };
 */
 
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Bg_Goron_Oyu_0x80B40080/func_80B40080.asm")
-
-#pragma GLOBAL_ASM("./asm/non_matchings/overlays/ovl_Bg_Goron_Oyu_0x80B40080/func_80B4009C.asm")
+void func_80B40100(BgGoronOyu* _this, GlobalContext* globalCtx);
+void func_80B400C8(BgGoronOyu* _this, GlobalContext* globalCtx);
+void func_80B401F8(BgGoronOyu* _this, GlobalContext* globalCtx);
+void func_80B40308(BgGoronOyu* _this, GlobalContext* globalCtx);
+void func_80B40394(BgGoronOyu* _this, GlobalContext* globalCtx);
+void func_80B40160(BgGoronOyu* _this, GlobalContext* globalCtx);
+
+void func_80B40080(BgGoronOyu* _this) {
```

Comment:
```suggestion
void func_80B40080(BgGoronOyu* this) {
```
Likewise as my comment above, I haven't seen `_this` used in actors, so for now it'd be more consistent to drop the underscore throughout the file